__tests__/app.test.js

```diff
@@ -169,3 +181,64 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe.only("POST /api/articles/:article_id/comments", () => {
+  test("201: POST /api/articles/:article_id/comments add comments when given article_id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const { comment } = response.body;
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          article_id: 1,
+          author: "butter_bridge",
+          body: "Loved this article!",
+          created_at: expect.any(String),
+          votes: 0,
+        });
+      });
+  });
+  test("400:POST/api/articles/:article_id/comments returns 400 when invalid article_id is given", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/invalid_id/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("400:POST/api/articles/:article_id/comments returns 400 when request body is missing required fields", () => {
+    const incompleteComment = {
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/invalid_id/comments")
+      .send(incompleteComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("404:POST/api/articles/:article_id/comments returns 404 when given an article_id that doesn't exist", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/invalid_id/comments")
+      .send(newComment)
+      .expect(400)
```

Comment:
you described your test to be testing for a 404 so why are we testing a 400 here ?